siege_utilities/geo/gazetteers/nominatim_gazetteer.py

```
@@ -0,0 +1,373 @@
+    # Gazetteer protocol
+    # ------------------------------------------------------------------
+
+    def lookup(
+        self,
+        name: str,
+        *,
+        country_hint: Optional[str] = None,
+        admin_hint: Optional[str] = None,
+    ) -> GazetteerResult:
+        if not name or not name.strip():
+            raise ValueError("NominatimGazetteer.lookup: empty name")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)